scripts/perf_mixed_workload.py

```
@@ -0,0 +1,353 @@
+        _validate_fixture_observation(plan, requested_shape, restored_observation)
+
+        result_values = {
+            "observed_shape": observed_shape,
+            "steady_samples": steady_samples,
+            "churn_samples": churn["samples"],
+            "latencies_ms": churn["latencies_ms"],
+            "throughput_ops_per_second": churn["throughput_ops_per_second"],
+            "render_observations": churn["render_observations"],
+            "failures": churn["failures"],
+        }
+    except BaseException as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `BaseException`

(BLE001)